contracts/modules/credit/SpigotedLine.sol

```diff
@@ -243,15 +237,16 @@ contract SpigotedLine is ISpigotedLine, LineOfCredit {
    * @return - whether or not spigot was released
   */
     function releaseSpigot() external returns (bool) {
-        return SpigotedLineLib.releaseSpigot(
-          address(spigot),
-          _updateStatus(_healthcheck()),
-          borrower,
-          arbiter
-        );
+        return
+            SpigotedLineLib.releaseSpigot(
```

Comment:
Can you make the return statement and function call on the same line please. Function args can still be spread across multiple lines.

Reply:
uhh yeah it's mostly my auto formatter

Reply:
gotcha not a problem, i noticed other places too. 
What are your formatter settings? I like your output better than the one I have setup in [`add-linting`](https://github.com/debtdao/smart-contracts/tree/add/linting/). Only reason i haven't merged it in is i dont like the output